src/app/global/app_signals.nim

```diff
@@ -9,3 +9,16 @@ type
     sectionType*: SectionType
 
 const TOGGLE_SECTION* = "toggleSection"
+## Emmiting this signal will turn on section/s with passed `sectionType` if that section type is
+## turned off, or turn it off in case that section type is turned on.
```

Comment:
nit: putting the comment above the line is more standard to the way we do it usually

Reply:
@jrainville I somehow agree and that's the way how is recommended in c++, but in nim all comments I've seen (but for procs/methods) were written beneath the declaration, and I was leaded by that logic, but in general as you say?